_posts/1-2-28-InlineGallery.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: doc
+permalink: /docs/ui-toolkit/components/inline-gallery
+title: Inline Gallery
+section: UI toolkit
+---
+
+# InlineGallery
+
+ImageGallery component renders a collection of `Image` components within `HorizontalPager`. Each preview component is rendered on a separate page.
+
+## InlineGallery
+![Inline Gallery example (left image)]({{ site.baseurl }}/img/ui-toolkit/image-gallery/image_gallery@2x.png "InlineGallery"){:.docs-component-image}
```

Comment:
Yeah, we don't have splitted images.. similar for horizontal pager / page indicators